### PR TITLE
#10841 Support export projects that aren't created by Robosaur

### PR DESCRIPTION
**Pull request by daniel-jeremia**
* Added `createdWith` and `projectFilter` in Config interface

* Added functionality in export project handler to handle stateless project export.

For further information please see `readme.md` in `Stateless project export` section

# Notes

Providing **only** `newestDate` in projectFilter will throw error. This should be fixed with this issue [#11171](https://github.com/vulcan-ai/vulcan/issues/11171)

### CODE REVIEW

**Comment by daniel-jeremia** [comment]
@darwin-gautalius @jonathancesario @ivanm-gdp is this safe to merge?

**Review by jonathancesario — APPROVED** [review]

**Review by ivanm-gdp — APPROVED** [review]
👍🏼 

I can export using the filters, a note though
doc-based quickstart uses `JSON_ADVANCED`, and I got unsupported format

from the UI, there are these four formats
image

**Comment by daniel-jeremia** [comment]
> doc-based quickstart uses `JSON_ADVANCED`, and I got unsupported format

oh right, I copied it from row-based quickstart before changing itu to CSV. I'll update it in this PR @ivanm-gdp

File: readme.md [hunk 1]
```diff
@@ -150,6 +150,55 @@ In multiple project creation using the `create-projects` command, the statefile
 In project export using `export-projects`, the JSON statefile is treated as source of truth. Only projects found in the statefile will be checked against the `statusFilter` and exported.  
 Robosaur will also record the project state when it was last exported, and subsequent runs will only export the project if there had been a forward change in the project status
 
+### Stateless project export
```
[the remark this answers is not in the document]
Comment (ivanm-gdp) [reply readme.md:155]: ohh, sorry wasn't clear yaa
table of contents

**Pull request merged**